dali/image/png.cc

```diff
@@ -16,12 +16,90 @@
 
 namespace dali {
 
-// Assume chunk points to a 4-byte value
-int ReadIntFromPNG(const uint8 *chunk) {
-  // reverse the bytes, cast
-  return (unsigned int) (chunk[0] << 24 | chunk[1] << 16 | chunk[2] << 8 | chunk[3]);
+namespace detail {
```

Comment:
```suggestion
namespace {
```
Since it's used only in this compilation unit, this can be an anonymous namespace. But it's a nitpicking ;)

Reply:
will do